README.md

```diff
@@ -8,28 +8,20 @@
 $ skywire-updater -h
 
 skywire-updater is responsible for checking for updates, and updating services
-associated with skywire.
-
-It takes one optional argument [config-path] which specifies the path to the
-configuration file to use. If no [config-path] is specified, the following 
-directories are searched in order:
-
-  1. /Users/anonymous/.skycoin/skywire-updater/config.yml
-  2. /usr/local/skycoin/skywire-updater/config.yml
+associated with skywire. Services to be updated will be based on a specified configuration file.
```

Comment:
Please reword to: `... based on specified configuration files.`

Reply:
Apologies for the late reply. Resolved.